src/state/slices/marketDataSlice/marketServiceManagerSingleton.ts

```diff
@@ -0,0 +1,24 @@
+// do not directly use or export, singleton
+import { MarketServiceManager } from '@shapeshiftoss/market-service'
+import { getConfig } from 'config'
+
+let _marketServiceManager: MarketServiceManager | undefined
+
+type GetMarketServiceManager = () => MarketServiceManager
+
+export const getMarketServiceManager: GetMarketServiceManager = () => {
+  const config = getConfig()
+  if (!_marketServiceManager) {
+    _marketServiceManager = new MarketServiceManager({
+      coinGeckoAPIKey: config.REACT_APP_COINGECKO_API_KEY,
+      // TODO(0xdef1cafe): market service manager needs to accept this into each method dynamically at runtime
+      yearnChainReference: 1,
```

Comment:
While at it, we could make this `CHAIN_REFERENCE.EthereumMainnet`, however that is currently a string meaning we would need to either cast it as number or update lib, obviously fine as is